packages/meditrak-server/src/apiV2/GETHandler/helpers.js

```diff
@@ -44,15 +43,15 @@ export const generateLinkHeader = (resource, pageString, lastPage, originalQuery
 };
 
 export const processColumnSelector = (models, unprocessedColumnSelector, baseRecordType) => {
-  if (unprocessedColumnSelector.includes('.')) {
-    const [resource, column] = unprocessedColumnSelector.split('.');
-    const recordType = resourceToRecordType(resource);
-    const model = models.getModelForDatabaseType(recordType);
-    const customSelector = model.customColumnSelectors && model.customColumnSelectors[column];
-    const selector = `${recordType}.${column}`;
-    return customSelector ? customSelector(selector) : selector;
-  }
-  return `${baseRecordType}.${unprocessedColumnSelector}`;
+  const [resource, column] = unprocessedColumnSelector.includes('.')
+    ? unprocessedColumnSelector.split('.')
+    : [baseRecordType, unprocessedColumnSelector];
+
+  const recordType = resourceToRecordType(resource);
+  const model = models.getModelForDatabaseType(recordType);
+  const customSelector = model?.customColumnSelectors?.[column];
+  const selector = `${recordType}.${column}`;
+  return customSelector ? customSelector(selector) : selector;
```

Comment:
This is a core change - it allows us to use `customColumnSelectors` regardless of whether the provided column specified a model name or not

Reply:
Nice cleanup too! The ternary with destructuring into array works really nicely